common.mk

```diff
@@ -518,7 +518,7 @@ optee-os-clean-common:
 optee-rust:
 ifeq ($(OPTEE_RUST_ENABLE),y)
 	@(cd $(OPTEE_RUST_PATH) && ./setup.sh)
-OPTEE_RUST_SET_ENV = source ~/.cargo/env &&
+PATH := $(PATH):$(HOME)/.cargo/bin
```

Comment:
I'd prefer `OPTEE_RUST_SET_ENV = PATH=$(PATH):$(HOME)/.cargo/bin`

Reply:
@jenswi-linaro Thanks for your advice.

```
PATH=$PATH:$HOME/.cargo/bin make OPTEE_RUST_ENABLE=y check-rust
```
works fine in the shell. 

But in makefile, neither
```
buildroot: optee-os optee-rust
    @$(OPTEE_RUST_SET_ENV) $(MAKE) -C ../out-br all
...
OPTEE_RUST_SET_ENV = PATH=$(PATH):$(HOME)/.cargo/bin
```
nor
```
buildroot: optee-os optee-rust
    @$(OPTEE_RUST_SET_ENV) $(MAKE) -C ../out-br all
...
OPTEE_RUST_SET_ENV = PATH=$$PATH:$$HOME/.cargo/bin
```
works.
Any suggestions? Thanks.

Reply:
> Any suggestions? Thanks.

Look at the log.
What command is really executed?
Try to add some debug prints to figure out what's going on.

Reply:
Oh, `OPTEE_RUST_SET_ENV = PATH=$(PATH):$(HOME)/.cargo/bin` set the PATH indeed,
But the `BR2_PACKAGE_OPTEE_RUST_EXAMPLES_TC_PATH_ENV` ( introduced by https://github.com/OP-TEE/build/pull/529) clear the PATH when building optee_rust_examples_ext:
```
BR2_PACKAGE_OPTEE_RUST_EXAMPLES_TC_PATH_ENV = $(PATH):$(ROOT)/toolchains/aarch64/bin
```
So I add PATH of cargo to:
```
BR2_PACKAGE_OPTEE_RUST_EXAMPLES_TC_PATH_ENV = $(PATH):$(ROOT)/toolchains/aarch64/bin:$(HOME)/.cargo/bin
```
and it works.

Reply:
Aha, that's better.